main.go

```diff
@@ -17,18 +16,12 @@ import (
 )
 
 func main() {
-	defaultPath, err := config.GetFilePath("")
-	if err != nil {
-		log.Errorln(log.Global, err)
-		os.Exit(1)
-	}
-
 	// Handle flags
 	var settings engine.Settings
 	versionFlag := flag.Bool("version", false, "retrieves current GoCryptoTrader version")
 
 	// Core settings
-	flag.StringVar(&settings.ConfigFile, "config", defaultPath, "config file to load")
+	flag.StringVar(&settings.ConfigFile, "config", "", "config file to load")
```

Comment:
It would be nice to know what config file is loaded; before it would state the full path: 

Loading config file C:\sup3rp4th\GoCryptoTrader\config.json..

Now with this PR:

Loading config file ..